packages/pigeon/platform_tests/ios_unit_tests/ios/RunnerTests/AllDatatypesTest.m

```diff
@@ -5,7 +5,7 @@
 #import <Flutter/Flutter.h>
 #import <XCTest/XCTest.h>
 #import "EchoMessenger.h"
-#import "all_datatypes.gen.h"
+#import "AllDatatypes.gen.h"
```

Comment:
The ObjC test changes are all:
- import renames for CamelCase files, or
- removal of prefixes.